_posts/2020-09-23-cytech-customer-story.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: 'Cytech uses Quarkus to save Greek Tourism'
+date: 2020-09-23
+tags: user-story
+synopsis: Cytech uses Quarkus to save Greek Tourism
+author: dandreadis
+---
+
+:imagesdir: /assets/images/posts/quarkus-user-stories/cytech
+
+= Cytech uses Quarkus to save Greek Tourism
+
+image::cytechLogoLarge.png[]
+
+== Tourism and Covid-19
+It's not new that the tourism industry is one the most affected industries from the Covid-19 pandemic. Hotels, restaurants, airlines, cruises are some of the businesses that had a really tough time this summer. Many countries, especially those whose economies rely on tourism, were looking for solutions in order to open up the borders for tourists while not  imposing their citizens to a great health risk. There are no magic solutions here but there is always room for doing your best to satisfy both requirements.
+
+The Greek government decided at some point to open up the borders in order to help the economy but it would be too reckless to do it without precautions and measures. That’s why they decided to build a system based on advanced technology that would choose the right travelers for Covid-19 testing. The idea is the following: if you cannot afford to test everyone then you should test those who are more likely to be positive to a Covid-19 test. This way the Greek state can use their resources wisely and at the same time save money for tests.
+
+The question is how. And this is the part that we entered the scene. The AI and machine learning algorithm was already proposed by a team of scientists. Its ambition was to cover exactly the need described above by efficiently choosing the right people to be tested. But these algorithms need data and at that point detailed data from the passengers entering the country were not available in any digital and organized form.
+
+
+== The basic idea
+The Greek government assigned our company, https://www.cytechmobile.com/[Cytech], to build a software that would gather the information from the passengers, filter it and pass it to all involved parties so they can do their job. One of them was the machine learning system called EVA. But the list with the parties needing this information is much longer: it includes the civil protection agency, the ministry of health, border agents, airlines etc. At the same time it would send an email to each of the passengers to inform them that their application is under review. When EVA would come up with the result it would send it back to us so we can email passengers the final document. Without it, their entrance in the country would not be permitted. To make a long story short, our job was to build a demanding https://travel.gov.gr[web form] to gather all passengers data but the most important part was to build a Data Broker capable of exchanging the right data with all the other information systems emphasizing on speed, robustness and resilience. 
```

Comment:
```suggestion
The Greek government assigned our company, https://www.cytechmobile.com/[Cytech], to build a software that would gather the information from the passengers, filter it and pass it to all involved parties so they can do their job. One of them was the machine learning system called EVA. But the list with the parties needing this information is much longer: it includes the civil protection agency, the ministry of health, border agents, airlines, etc. At the same time, it would send an email to each of the passengers to inform them that their application is under review. When EVA would come up with the result, it would send it back to us to email passengers the final document. Without it, their entrance into the country would not be permitted. To make a long story short, our job was to build a demanding https://travel.gov.gr[web form] to gather all passenger's data. Still, the most important part was to create a Data Broker capable of exchanging the right data with all the other information systems emphasizing speed, robustness, and resilience. 
```